From c3ff5a8da6348439b4f1d5b67dd4ac861252b3b9 Mon Sep 17 00:00:00 2001
From: Johan Hovold <johan+linaro@kernel.org>
Date: Thu, 16 Nov 2023 16:38:46 +0100
Subject: [PATCH 001/173] of: fix recursion typo in kernel doc

Fix a typo in the kernel doc for the of_platform_depopulate() functions,
which remove children "recursively".

Signed-off-by: Johan Hovold <johan+linaro@kernel.org>
---
 drivers/of/platform.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/of/platform.c b/drivers/of/platform.c
index 126d265aa7d8..2293059758d1 100644
--- a/drivers/of/platform.c
+++ b/drivers/of/platform.c
@@ -668,7 +668,7 @@ EXPORT_SYMBOL_GPL(of_platform_device_destroy);
  * @parent: device which children will be removed
  *
  * Complementary to of_platform_populate(), this function removes children
- * of the given device (and, recurrently, their children) that have been
+ * of the given device (and, recursively, their children) that have been
  * created from their respective device tree nodes (and only those,
  * leaving others - eg. manually created - unharmed).
  */
@@ -737,7 +737,7 @@ static int devm_of_platform_match(struct device *dev, void *res, void *data)
  * @dev: device that requested to depopulate from device tree data
  *
  * Complementary to devm_of_platform_populate(), this function removes children
- * of the given device (and, recurrently, their children) that have been
+ * of the given device (and, recursively, their children) that have been
  * created from their respective device tree nodes (and only those,
  * leaving others - eg. manually created - unharmed).
  */
-- 
2.43.0

